.devcontainer/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# Dependencies versions
+ARG PYTHON_VERSION="3.7.10"
+
+FROM python:${PYTHON_VERSION}-buster
+
+# Dependencies versions
+ARG DOCKER_COMPOSE_VERSION="1.29.2"
+ARG COOKIECUTTER_VERSION="1.7.3"
+ARG AZURE_CLI_VERSION="2.24.0-1~buster"
+ARG NODE_VERSION="15.x"
+ARG DOTNET_VERSION="5.0"
+ARG JDK_VERIOSN="2:1.11-71"
+ARG MAVEN_VERIOSN="3.6.0-1"
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends build-essential && \
+    apt-get install -y python2.7-dev libffi-dev libssl-dev
```

Comment:
>python2.7-dev [](http://example.com/codeflow?start=23&length=13)

python 2.7 is no longer supported in iotedgedev

Reply:
Ah! I'll remove it from the contribution docs too then ;)

Reply:
I've also updated the doc with devcontainer docs. Let me know if that's not OK